uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -46,6 +47,62 @@ var Promise = require('es6-promise').Promise;
     });
   });
 
+  /**
+   * Constructs URL for BackstopJS scenario based on
+   * site URL, scenario config URL and contact "roles" and IDs map
+   *
+   * @param  {String} siteUrl
+   * @param  {String} scenarioUrl
+   * @param  {Object} contactIdsByRoles
+   * @return {String}
+   */
+  function constructBackstopJSScenarioUrl (siteUrl, scenarioUrl, contactIdsByRoles) {
+    return siteUrl + '/' +
```

Comment:
sorry, this is not what i meant. What i meant was the concatenation + replacement one liner was too much. First the replacement and then the concatenation. I don't think this function is necessary, but i'll leave it up to you

Reply:
@AkA84 , @deb1990 asked to move it to another function. So I leave it.

Reply:
Done.